src/components/playback/mediasession.js

```diff
@@ -261,8 +202,8 @@ define(['playbackManager', 'nowPlayingHelper', 'events', 'connectionManager'], f
     function execute(name) {
         playbackManager[name](currentPlayer);
     }
-    if (navigator.mediaSession) {
 
+    if (navigator.mediaSession) {
```

Comment:
why're you checking it differently? in code above you used `if 'mediaSession' in navigator`

Reply:
This was an original line that I moved down. I'll change the check too, for consistency.